presto-cache/src/main/java/com/facebook/presto/cache/CachingFileSystem.java

```diff
@@ -50,28 +50,11 @@
 public final class CachingFileSystem
```

Comment:
I don't think we need to change much of this class except renaming it to LocalRangeCachingFileSystem

Reply:
This file is now a delegation layer to the actual caching file systems. The original class is renamed to BasicCachingFileSystem.

Reply:
We don't need CachingFileSystem then. Otherwise, CachingFileSystem is nothing but a complete delegate of ExtendedFileSystem

Reply:
You can resolve this, we already discussed in other thread.